tests/framework/extensions/provisioning/verify.go

```diff
@@ -44,6 +44,7 @@ const (
 	hostnameLimit               = 63
 	machineNameAnnotation       = "cluster.x-k8s.io/machine"
 	machineSteveResourceType    = "cluster.x-k8s.io.machine"
```

Comment:
I know this is just been in there but these constants in line 45 and 46 point to the same string, maybe change it to one?

Reply:
ah missed that part, thanks for clarifying